pkg/build/builder/dockerutil.go

```diff
@@ -110,8 +110,15 @@ func pullImage(client DockerClient, name string, authConfig docker.AuthConfigura
 	logProgress := func(s string) {
 		glog.V(0).Infof("%s", s)
 	}
+	tag := "latest"
```

Comment:
@mfojtik is there a parser for docker pull specs that will give me the tag?  @csrwng thought there was but we didn't immediatley find it.  ParseDockerImageReference isn't as clean as i'd like.